BUILD.bazel

```diff
@@ -929,13 +947,18 @@ cc_library(
         ":interpreter_ops",
         ":stablehlo_ops",
         ":stablehlo_ops_inc_gen",
+        ":broadcast_utils",
+        ":chlo_rewriters_inc_gen",
```

Comment:
This section needs to be sorted, will fix buildifier CI lint:

```
        ":base",
        ":broadcast_utils",
        ":chlo_ops",
        ":chlo_rewriters_inc_gen",
        ":interpreter_ops",
```

Reply:
Sorry there were 2 errors in the patch I sent, luckily all the other CI is passing so should be good for submit after these fixes!